san.cnf

```diff
@@ -0,0 +1,22 @@
+[req]
+default_bits = 4096
+prompt = no
+default_md = sha256
+distinguished_name = dn
+req_extensions = req_ext
+
+[dn]
+C=UnitedStates
```

Comment:
The country is a two letter code not the whole name, This causes an error on my machine.

Reply:
Whoops.

Everything should be fixed, and the new crt and key are on the dev server (https://postebackend.duckdns.org)

Give it a try on your branch and let me know if it works